staff_features/accessions/accession_spawn_digital_object.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Accession spawn Digital Object
+  Background:
+    Given an administrator user is logged in
+      And the "Spawn description for Digital Object instances from linked record" setting is enabled in the Repository Preferences
+      And an Accession has been created
```

Comment:
a small suggestion here: as the final assertion checks the Title, Languages and Dates, perhaps it would be more accurate to add a Given step about the Accession fields:
And the Accession has a Title, Languages and Dates